ironfish-cli/src/commands/wallet/import.ts

```diff
@@ -88,11 +88,11 @@ export class ImportCommand extends IronfishCommand {
   }
 
   async importTTY(): Promise<AccountImport> {
-    const accountName = await CliUx.ux.prompt('Enter the account name', {
+    const accountName = await CliUx.ux.prompt('Enter the wallet name', {
       required: true,
     })
 
-    const spendingKey = await CliUx.ux.prompt('Enter the account spending key', {
+    const spendingKey = await CliUx.ux.prompt('Enter the wallet spending key', {
```

Comment:
This is incorrect: this command is to import a single account into a wallet, which contains multiple accounts.